images/router/haproxy/Dockerfile

```diff
@@ -5,7 +5,9 @@
 #
 FROM openshift/origin
 
-RUN INSTALL_PKGS="haproxy" && \
+RUN INSTALL_PKGS="haproxy18" && \
+    yum install yum-config-manager && \
+    yum-config-manager --add http://cbs.centos.org/repos/paas7-openshift-origin38-testing/x86_64/os/ && \
```

Comment:
Why is the haproxy18 package in the normal set of dependencies we already set?  We spent 2 releases trying to stop doing it.

Reply:
I assume you meant "why ISN'T the haproxy18..."